config/initializers/wrap_parameters.rb

```diff
@@ -5,10 +5,10 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json] if respond_to?(:wrap_parameters)
+    wrap_parameters format: [:json]
 end
 
 # To enable root element in JSON for ActiveRecord objects.
 ActiveSupport.on_load(:active_record) do
-  self.include_root_in_json = false
+    self.include_root_in_json = true
```

Comment:
Aha! I think this may have been responsible for our JSON error. But no need to change anything!

Reply:
Yeah correct @jywarren :smile: